02.lovely-movies/styles/styles.css

```diff
@@ -74,6 +74,10 @@ body.loading .loader {
     align-items: center;
     box-shadow: 0 1px 10px 2px rgba(0, 0, 0, 0.3);
     height: 70px;
+    position: sticky;
+    top: 0;
+    z-index: 5;
```

Comment:
I think `1` would be enough as the value.

Reply:
but I added it in .main-header not  the body, is there some thing I don't understand.
<img width="349" alt="image" src="https://user-images.githubusercontent.com/95933671/211213145-b52d3083-9d6f-4483-bbfd-c9d449b06cce.png">

Reply:
Sorry, may bad!